### ISSUE TITLE
[PresentationAPI] Controller IDL harness tests fail to set navigator.presentation.defaultRequest

### ISSUE THREAD

**Issue by mfoltzgoogle** [w3c/web-platform-tests#3294]
In Chrome 53, the controller IDL harness tests [1] for navigator.presentation.defaultRequest are failing because they fail to set a defaultRequest.  I can look into how to set this value appropriately for this test case.

@louaybassbouss

```
Fail	Stringification of navigator.presentation.defaultRequest	assert_equals: class string of navigator.presentation.defaultRequest expected "[object PresentationRequest]" but got "[object Null]"
    at IdlObject.<anonymous> (http://localhost:8000/resources/idlharness.js:1494:9)
    at Object.Test.step (http://localhost:8000/resources/testharness.js:1403:25)
    at test (http://localhost:8000/resources/testharness.js:501:18)
    at IdlObject.IdlInterface.test_primary_interface_of (http://localhost:8000/resources/idlharness.js:1490:5)
    at IdlObject.IdlInterface.test_object (http://localhost:8000/resources/idlharness.js:1440:10)
Fail	PresentationRequest interface: navigator.presentation.defaultRequest must inherit property "start" with the proper type (0)	Cannot use 'in' operator to search for 'hasOwnProperty' in null
TypeError: Cannot use 'in' operator to search for 'hasOwnProperty' in null
    at http://localhost:8000/resources/testharness.js:1134:37
    at IdlObject.<anonymous> (http://localhost:8000/resources/idlharness.js:1546:25)
    at Object.Test.step (http://localhost:8000/resources/testharness.js:1403:25)
    at test (http://localhost:8000/resources/testharness.js:501:18)
    at IdlObject.IdlInterface.test_interface_of (http://localhost:8000/resources/idlharness.js:1540:13)
    at IdlObject.IdlInterface.test_object (http://localhost:8000/resources/idlharness.js:1452:27)
    at .<anonymous> (http://localhost:8000/resources/idlharness.js:403:36)
    at Array.forEach (native)
    at IdlArray.test (http://localhost:8000/resources/idlharness.js:401:32)
    at http://localhost:8000/presentation-api/controlling-ua/idlharness.html:121:19
```

**Comment by louaybassbouss** [comment]
Thx @mfoltzgoogle I am back from vacation please let me know if I can help on this

**Comment by louaybassbouss** [comment]
We can close this issue since it is fixed in PR #3533
